nixos/modules/config/system-path.nix

```diff
@@ -132,28 +133,15 @@ in
       # outputs TODO: note that the tools will often not be linked by default
       postBuild =
         ''
-          if [ -x $out/bin/update-mime-database -a -w $out/share/mime ]; then
-              XDG_DATA_DIRS=$out/share $out/bin/update-mime-database -V $out/share/mime > /dev/null
-          fi
-
           if [ -x $out/bin/gtk-update-icon-cache -a -f $out/share/icons/hicolor/index.theme ]; then
```

Comment:
This should probably be moved to `xdg.icons`

Reply:
Hm, I wasn't sure about that - I thought maybe there should be a GTK module that does this?

Reply:
Right, that looks like GTK thing: https://developer.gnome.org/gtk3/stable/gtk-update-icon-cache.html

We should probably get a module later.

Reply:
Similarly for the glib schemas.